test/parallel/test-cli-node-options.js

```diff
@@ -67,6 +67,7 @@ if (common.hasCrypto) {
 // V8 options
 expect('--abort_on-uncaught_exception', 'B\n');
 expect('--disallow-code-generation-from-strings', 'B\n');
+expect('--jitless', 'B\n');
```

Comment:
```suggestion
if (!common.isAIX)
  expect('--jitless', 'B\n');
```

Reply:
`node --jitless` coredumps on AIX in v12.x-staging but is okay on master. @nodejs/platform-aix is there something backportable to V8 7.8 to address this?

Reply:
@richardlau I'm going to land this and I've opened a tracking issue to fix this https://github.com/nodejs/node/issues/32600